contrib-docs/community-guidelines/index.md

```diff
@@ -1,30 +1,30 @@
-# Contribute
+# Community guidelines
 
-_[draft]_
-
-Welcome to MDN Web Docs on Github! MDN Web Docs is ... 
-
-MDN Web Docs is an open source project. There are lots of different ways you can contribute
-
-_(see community activity stuff for above)_
-
-Here we hold our guidelines and how tos for contributing, and what to expect from working with us on github (coc, pr ettiquette etc...)
+👋 Welcome to MDN Web Docs on GitHub! MDN Web Docs is an open-source, collaborative project that documents web platform technologies, including [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML), [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS), [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), and [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API). We also provide extensive [learning resources](https://developer.mozilla.org/en-US/docs/Learn) for early-stage developers and students.
 
 ## Ways to contribute
 
-- (Learn Forum)[linktolearnforum] - help users in our learn forum, see the guidelines on how to here
-- (Translated Content)[linktotranslatedcontent] - help translate MDN Web Docs into one of our active locales (list locales)
-- (Help with updating content or platform improvements by contributing to our repositories)[listofrepositories]
-	- issues
-	- pr reviews
+- [Fixing MDN Web Docs content bugs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Fixing_MDN_content_bugs)
+- [Reviewing MDN Web Docs edits](https://github.com/mdn/content/blob/main/REVIEWING.md)
+- [Help beginners to learn on MDN Web Docs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Help_beginners)
+- [Contribute to MDN Web Docs interactive examples](https://github.com/mdn/interactive-examples/blob/main/CONTRIBUTING.md)
+- [Help translate MDN Web Docs](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Localize)
+- [Help fix known platform issues](https://github.com/mdn/yari/issues)
+- [Help us keep browser compatibility data up to date.](https://github.com/mdn/browser-compat-data)
```

Comment:
To note these links will need updating

Reply:
Actually maybe we discuss because I think 'Fixing issues' which can include all repos...